test/test_torch.py

```diff
@@ -3703,6 +3703,25 @@ def test_masked_fill_bool_tensor(self, device):
         dst = dst.masked_fill(mask, False)
         self.assertEqual(dst, torch.tensor([True, False, True], device=device))
 
+    # masked_fill() ref used to use item(), make sure the reported repro works.
+    # Not an OpInfo test because it breaks TestMathBitsCUDA::test_conj_view.
+    # https://github.com/pytorch/pytorch/issues/81018 (repro)
+    # https://github.com/pytorch/pytorch/pull/82737 (context)
+    @skipIfRocm
+    @onlyCUDA  # we specifically need device='cuda' for this one
+    def test_masked_fill_device_promotion(self, device):
```

Comment:
why can't we add this reference input to OpInfo? TestMathBits doesn't run on reference inputs.

Reply:
Ah, I didn't know that! I've only tried putting it in sample inputs. Will do!